smithy-model/src/main/java/software/amazon/smithy/model/shapes/SmithyIdlModelSerializer.java

```diff
@@ -302,6 +309,16 @@ public Builder basePath(Path basePath) {
             return this;
         }
 
+        /**
+         * Enables serializing shapes in the Smithy prelude.
+         * Defaults to false.
+         * @return Returns the builder.
+         */
+        public Builder enableSerializingPrelude() {
```

Comment:
nit: I think something like `serializePrelude` would be a much more succinct name

Reply:
fixed in 28dc5ac9b7879f02a778303931c750541b04ce98.